### PR TITLE
put  -(dash) in multiword option names  # issue 242, 

### PR DESCRIPTION
**Pull request by KSchopmeyer**
This pr is is built on pr #244.
Changed a number of the multiword option names to include - to separate words.

See commit message for details

### CODE REVIEW

File: pywbemtools/pywbemcli/_cmd_class.py [hunk 1]
```diff
@@ -175,10 +175,10 @@ class hierarchy defined by `CLASSNAME` argument if provided.
 
     The output format is defined by the output-format general option.
 
-    The includeclassqualifiers, includeclassorigin options define optional
+    The local-only and no-qualifiers options define optional
```
Comment (andy-maier) [inline pywbemtools/pywbemcli/_cmd_class.py]: Show option names local-only and no-qualifiers with leading dashes and in example font: `--local-only` and `--no-qualifiers`.

File: pywbemtools/pywbemcli/_cmd_class.py [hunk 2]
```diff
@@ -175,10 +175,10 @@ class hierarchy defined by `CLASSNAME` argument if provided.
...
     information to be included in the output.
 
-    The deepinheritance option defines whether the complete hiearchy is
+    The deep-inheritance option defines whether the complete class hiearchy is
```
Comment (andy-maier) [inline pywbemtools/pywbemcli/_cmd_class.py]: Show option name deep-inheritance with leading dashes and in example font: `--deep-inheritance`.

File: pywbemtools/pywbemcli/_cmd_class.py (hunk 1 above)
Comment (andy-maier) [inline pywbemtools/pywbemcli/_cmd_class.py]: Show option names local-only and no-qualifiers with leading dashes.

File: pywbemtools/pywbemcli/_cmd_class.py (hunk 2 above)
Comment (andy-maier) [inline pywbemtools/pywbemcli/_cmd_class.py]: Show option name deep-inheritance with leading dashes.

File: pywbemtools/pywbemcli/_cmd_instance.py [hunk 3]
```diff
@@ -58,12 +58,29 @@
...
+                      'returned instance. Some servers may not process this '
+                      'parameter.')]
+
+
+localonlyenum_option = [              # pylint: disable=invalid-name
+    click.option('-l', '--local-only', is_flag=True, required=False,
+                 help='Show only local properties of the instances. This '
+                      'subcommand may use either pull or traditional '
+                      'operations depending on the server and the '
+                      '"--use--pull-ops" general option. If pull operations '
```
Comment (andy-maier) [inline pywbemtools/pywbemcli/_cmd_instance.py]: Show the "--use--pull-ops" option without quotes around it, and remove the extra dash inside of the name.

File: pywbemtools/pywbemcli/_cmd_server.py [hunk 4]
```diff
@@ -33,6 +33,10 @@ def server_group():
     """
     Command Group for WBEM server operations.
 
+    The server comman-group defines commands to inspect and manage core
```
Comment (andy-maier) [inline pywbemtools/pywbemcli/_cmd_server.py]: Typo: "comman".

File: tests/unit/test_class_subcmd.py [hunk 5]
```diff
@@ -106,31 +106,32 @@ class hierarchy defined by `CLASSNAME` argument if provided.
 
   The output format is defined by the output-format general option.
 
-  The includeclassqualifiers, includeclassorigin options define optional
+  The local-only and no-qualifiers options define optional
```
Comment (andy-maier) [inline tests/unit/test_class_subcmd.py]: Show option names local-only and no-qualifiers with leading dashes.

File: tests/unit/test_class_subcmd.py [hunk 6]
```diff
@@ -106,31 +106,32 @@ class hierarchy defined by `CLASSNAME` argument if provided.
...
   information to be included in the output.
 
-  The deepinheritance option defines whether the complete hiearchy is
+  The deep-inheritance option defines whether the complete class hiearchy is
```
Comment (andy-maier) [inline tests/unit/test_class_subcmd.py]: Show option name deep-inheritance with leading dashes.

File: pywbemtools/pywbemcli/_cmd_class.py (hunk 1 above)
Comment (KSchopmeyer) [reply pywbemtools/pywbemcli/_cmd_class.py]: DONE

File: pywbemtools/pywbemcli/_cmd_class.py (hunk 2 above)
Comment (KSchopmeyer) [reply pywbemtools/pywbemcli/_cmd_class.py]: DONE

File: pywbemtools/pywbemcli/_cmd_instance.py (hunk 3 above)
Comment (KSchopmeyer) [reply pywbemtools/pywbemcli/_cmd_instance.py]: Agreed. DONE

File: pywbemtools/pywbemcli/_cmd_server.py (hunk 4 above)
Comment (KSchopmeyer) [reply pywbemtools/pywbemcli/_cmd_server.py]: DONE

File: tests/unit/test_class_subcmd.py (hunk 5 above)
Comment (KSchopmeyer) [reply tests/unit/test_class_subcmd.py]: DONE.

File: tests/unit/test_class_subcmd.py (hunk 6 above)
Comment (KSchopmeyer) [reply tests/unit/test_class_subcmd.py]: Fixed in _cmd_class.py

**Comment by andy-maier** [comment]
Approved, but since it is on top of PR #244, let's first get 244 in and then rebase this one again.

**Pull request merged**